spirv_cross.cpp

```diff
@@ -1126,6 +1126,10 @@ void Compiler::set_member_decoration(uint32_t id, uint32_t index, Decoration dec
 		dec.location = argument;
 		break;
 
+	case DecorationComponent:
```

Comment:
Hm, this isn't implemented in GLSL or HLSL either. Guess I'll implement GLSL after this PR goes through. Not sure if HLSL supports this?